src/features/events/components/EventsList/index.tsx

```diff
@@ -1,6 +1,6 @@
 import type { FC } from 'react'
 
-import { EventItemContainer } from '~/features/events/components/EventItemContainer'
+import { EventItemComponent } from '~/features/events/components/EventItemComponent'
```

Comment:
And well, turns out the `EventItemComponent` was already quite smart, so no need to have a container wrapping it. Or, we could refactor connectors to move to the container.